Add section on different ways to authenticate to the Prometheus API (#1718)
superfly/docs: monitoring/metrics.html.md

### External or self-hosted Grafana

> You can also configure your Prometheus endpoint with an existing Grafana installation, or [host one on Fly.io](https://github.com/fly-apps/grafana). Either way, you set it up thus:

Reviewer: ```suggestion
You can also configure your Prometheus endpoint with an existing Grafana installation, or [host one on Fly.io](https://github.com/fly-apps/grafana). Either way, you can set it up like this:
```

Instrument your app and expose your metrics on `0.0.0.0`.

There are many supported [client libraries](https://prometheus.io/docs/instrumenting/clientlibs/) as well as off-the-shelf [exporters](https://prometheus.io/docs/instrumenting/exporters/) able to return Prometheus-formatted metrics.

## Authentication

Authenticating to the Prometheus API can be achieved a few different ways, depending on the level of access you want your token to have.

### Fly Access Token

As in the earlier example, a full access token can be generated with `flyctl auth token` and then passed as a bearer token in the `Authorization` header. The header looks like:

```
Authorization: Bearer THE_TOKEN
```

### Fly org-restricted or read-only token

This kind of token or "[macaroon](https://fly.io/blog/macaroons-escalated-quickly/)" can be scoped to a single organization or configured to only allow read operations, which can be safer than using a full-blown read-write token that grants access to all organizations under your account.

> ### Generating

Reviewer: ```suggestion
### Generating tokens
```

> An org-restricted token can be generated like so:

Reviewer: ```suggestion
Create an org-restricted token:
```

```
fly token create org -o THE_ORGANIZATION
```

> A read-only token is created like this:

Reviewer: ```suggestion
Create a read-only org-restricted token:
```

```
fly token create readonly
```

> Note that the read-only token still has access to all organizations in your account (although it cannot modify anything, only read data). These two can be combined to create a read-only token that is restricted to a single organization:

Reviewer: The read-only token you create with `fly token create readonly` is already an org scoped token, so you don't need to do this.

Author: Thanks, I'll remove this part entirely :)

```
fly token create readonly --from-existing -t "$(fly token create org -o THE_ORGANIZATION)"
```

These tokens look like this once generated: `FlyV1 fm2_lJPECAAAAAAAAC7txBAzYI6PRWhHLT...(a lot of base64-encoded text)`.

> #### Using

Reviewer: ```suggestion
#### Using tokens
```
